src/frontends/tensorflow_common/src/op/reduce.cpp

```diff
@@ -22,13 +22,28 @@ namespace op {
 
 template <typename T>
 OutputVector translate_direct_reduce_op(const NodeContext& node) {
-    default_op_checks(node, 2, {"Any", "All", "EuclideanNorm", "Max", "Mean", "Min", "Prod", "Sum"});
+    default_op_checks(node, 2, {"Any", "All", "EuclideanNorm", "Max", "Mean", "Min", "Prod", "Sum"}, true);
```

Comment:
we should create separate translator for `Prod` since this moment. That is because other operations in this list like `Any`, `All`, `Min` are not supported for complex type. So let us decouple conversion for `Prod`

Reply:
@rkazants should i create the translate_prod_op inside the same file reduce.cpp or create a new file for the translator

Reply:
please revert this change on this code line